### PR TITLE
Netlify

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
README and deployment

### Description
This PR
- adds the netlify badge to the intro.md if it get's deployed using netlify.
- adds a copy of our code of conduct from the main repository which is necessary for the netlify open source sponsoring.

### Steps

- [x] Added implementation
- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: netlify/patch-intro.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+const { copyFile } = require('fs/promises');
+
+console.log("Copy patched intro.md")
+copyFile("netlify/intro.md", "public/mock-backend/public/intro.md")
+  .then(() => console.log("Done!"))
```
Comment (ErikMichelson) [inline netlify/patch-intro.js:11]:
```suggestion
  .then(() => console.log("Updated intro.md content to include netlify banner"))
```

File: README.md [hunk 2]
```diff
@@ -72,3 +72,9 @@ This will build the app in production mode and save it into the `.next` folder.
 best performance, minimized and the filenames include a hash value of the content. Don't edit them by hand!
 
 You can run the production build using the built-in server with `yarn start`.
+
+## UI Test
+
+Curious about the new look and feel? We provide a demo of the new UI on [hedgedoc.dev](https://hedgedoc.dev). This version uses mocked data and has no data persistence.
```
Comment (davidmehren) [inline README.md:78]:
```suggestion
Curious about the new look and feel? We provide a demo of the new UI on [hedgedoc.dev](https://hedgedoc.dev). This version uses mocked data and has no data persistence.
The UI demo is hosted on Netlify, their [privacy policy](https://www.netlify.com/privacy) applies.
```
Maybe also add a note to https://hedgedoc.org/privacy-policy/?

Comment (ErikMichelson) [reply README.md:78]: I like the idea of adding the netlify privacy link. However, I don't see how this is related to hedgedoc.org/privacy-policy.

**Review by ErikMichelson — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mrdrogdrog) [reply README.md:78]: tbh I would just link our privacy policy and add netlify to it. I mean. We list every service we use there.

Comment (mrdrogdrog) [reply README.md:78]: See https://github.com/hedgedoc/hedgedoc.github.io/pull/202

**Pull request merged**